lightning_talk/models/post.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.conf import settings
+from .upvote import Upvote
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=40)
+    description = models.CharField(max_length=200)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
+    upvotes = models.ManyToManyField(Upvote, blank=True)
```

Comment:
I think there is no need to make Post <> Upvote relation ManyToMany. One upvote will only belong to single Post. Can you update to use ForeignKey relation instead? Or was there some other reason to use ManyToMany?